examples/text_labels_and_annotations/font_file.py

```diff
@@ -4,8 +4,8 @@
 ===================================
 
 Although it is usually not a good idea to explicitly point to a single ttf file
-for a font instance, you can do so using the `.font_manager.FontProperties`
-*fname* argument.
+for a font instance, you can do so by passing a `pathlib.Path` instance as the
+*font* parameter.
```

Comment:
Should mention that strings are intentionally not supported, but you can simply wrap them in a `Path` if necessary.

Reply:
done